app/src/main/java/org/simple/clinic/drugs/selectionv2/dosage/DosagePickerEvents.kt

```diff
@@ -4,3 +4,5 @@ import org.simple.clinic.widgets.UiEvent
 import java.util.UUID
 
 data class DosagePickerSheetCreated(val drugName: String, val patientUuid: UUID) : UiEvent
+
+data class DosageItemClicked(val dosage: DosageOption) : UiEvent
```

Comment:
Can you rename the property `dosage` -> `dosageOption` since it can also be a `None`?